src/Caching/ValueObject/Storage/FileCacheStorage.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Symplify\EasyCodingStandard\Caching\ValueObject\Storage;
 
+use Nette\Utils\FileSystem as UtilsFileSystem;
```

Comment:
@vasilvestre for manual test, with directly change on vendor, change it with prefixed use statement:

```php
use ECSPrefix202405\Nette\Utils\FileSystem as UtilsFileSystem;
```

Reply:
I will try soon

Reply:
Signature is invalid for FileSystem in construct